foca/foca.py

```diff
@@ -0,0 +1,57 @@
+"""FOCA class definition"""
+
+from foca.api.register_openapi import (register_openapi, OpenAPIConfig)
+from foca.config.config_parser import (get_conf, get_conf_type)
+from foca.config.log_config import configure_logging
+from foca.database.register_mongodb import register_mongodb
+from foca.errors.errors import register_error_handlers
+from foca.factories.connexion_app import create_connexion_app
+from foca.security.cors import enable_cors
+from connexion import App
+from foca.config.config_handler import Config
+from typing import Iterable
+
+
+class FOCA():
+    """Description"""

```

Comment:
As this is the main entry point into FOCA's functionality, please add a nice [Google-style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for it, similar to what was being done for the `foca.api.register_openapi` module.